ode-wrapper/src/main/java/com/trihydro/odewrapper/controller/WydotTimRcController.java

```diff
@@ -148,6 +151,9 @@ public ResponseEntity<String> submitAllClearRoadConditionsTim(@RequestBody TimRc
         }
 
         String responseMessage = gson.toJson(resultList);
+        if (errList.size() > 0) {
+            utility.logWithDate("Failed to send TIMs: " + gson.toJson(errList), this.getClass());

```

Comment:
Suggestion (non-blocking): Add `@Slf4j` annotation to class if it isn't already present & transition to using `log.info`, `log.debug`, `log.error` instead of `utility.logWithDate`.